TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/console_api/settings.py

```diff
@@ -28,8 +28,23 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-DEPLOYED_STAGE = os.environ.get('MIGRATION_STAGE')
-ALLOWED_HOSTS = ['migration-console', f'migration-console.migration.{DEPLOYED_STAGE}.local', 'localhost']
+
+def get_allowed_hosts():
```

Comment:
minor, but a type annotation would help make it a bit more clear what's going on here.

Reply:
Added